src/app/character-tile/StoreIcons.tsx

```diff
@@ -1,27 +0,0 @@
-import ClassIcon from 'app/dim-ui/ClassIcon';
-import { DimStore } from 'app/inventory/store-types';
-import React from 'react';
-import styles from './StoreIcons.m.scss';
-
-/**
- * Show both the store emblem and class icon for a given store.
- */
-export function StoreIcons({ store, useBackground }: { store: DimStore; useBackground?: boolean }) {
```

Comment:
For historical reference, this was called `StoreIcons` because it displays 2 icons - emblem and class.

Reply:
this makes perfect sense